### DISCUSSION TITLE
Installation of eslint-plugin-import in ESLint

### DISCUSSION THREAD

**Discussion by Taiki0807** [eslint/eslint#17191] [Q&A] [answered]
I had been using ESLint without installing eslint-plugin-import, but I was still getting import-related errors and it seemed to be working fine. Why is it working without installing it?

When I run npm run lint, I get the following error:
```
/english-learning-frontend/app/components/pages/SignUp/SignUp.tsx
4:1  error  `./SignUp.module.css` import should occur before import of `@/app/components/features/LoginForm/AuthContext`  import/order
```

package.json
[https://github.com/Taiki0807/english-learning-frontend/blob/main/package.json](https://github.com/Taiki0807/english-learning-frontend/blob/main/package.json)

.eslintrc.json
[https://github.com/Taiki0807/english-learning-frontend/blob/main/.eslintrc.json](https://github.com/Taiki0807/english-learning-frontend/blob/main/.eslintrc.json)

**Comment by sam3k** [comment]
Hi @Taiki0807 can you provide a reproducible demo? I've looked at your repo and I don't see a file named `SignUp.tsx`. My wild guess is that you have a lint rule to that requires you import the css files before the  `AuthContext`. This is just a warning and may work most of the times. 

To fix, you would just move the `./SignUp.module.css` import above the `AuthContext` import.

**Reply by Taiki0807** [reply]
@sam3k 
Thank you for replying.
I was wondering because there was no mention of eslint-plugin-import in the package.json file, but it was still functioning. Is it not necessary to run "npm install eslint-plugin-import --save-dev" if it is already installed as a dependency?

**Reply by sam3k** [reply]
Oh I see. The rule is probably coming from one of [these two](https://github.com/Taiki0807/english-learning-frontend/blob/main/.eslintrc.json#L6-L7) plugins imported in `.eslintrc.json`.

**Reply by Taiki0807** [reply]
npm ls eslint-plugin-import

Here are the results:
next.js-template@0.1.0 /english-learning-frontend
└─┬ eslint-config-next@13.3.4
  ├─┬ eslint-import-resolver-typescript@3.5.3
  │ └── eslint-plugin-import@2.27.5 deduped
  └── eslint-plugin-import@2.27.5

If it is already installed as a dependency, is it not necessary to run "npm install eslint-plugin-import --save-dev"?

**Reply by sam3k** [reply, marked as answer]
I don't think is necessary to install in your case as the `next.js-template` module already has it.

**Reply by Taiki0807** [reply]
Ok.
Thank you for your support, I appreciate it.
